services/bb-pacman/scripts/local-development/.env-example

```diff
@@ -0,0 +1,113 @@
+# ------------------------------------------------------------
```

Comment:
Do you think these multi-service components should be inside the bb-pacman folder? I kinda figure they should be a higher level. If you like them here, let's delete the /docker/bbp-network folder at the repo root.

Reply:
nope. seems to me bbp-network is good for demos etc, these are for active development.